### PR TITLE
Remove Unused Style code

### PR DESCRIPTION
**Pull request by dvaldivia**
Signed-off-by: Daniel Valdivia <18384552+dvaldivia@users.noreply.github.com>

### CODE REVIEW

**Review by kaankabalak — DISMISSED** [review]

**Review by allanrogerr — APPROVED** [review]

**Comment by kaankabalak** [comment]
Hi @dvaldivia, we seem to have an extra `linkClass` lying around on portal-ui/src/screens/Console/Support/utils.tsx:107

File: portal-ui/src/screens/Console/Support/Register.tsx [hunk 1]
```diff
@@ -76,17 +76,13 @@ interface IRegister {
 
 const styles = (theme: Theme) =>
   createStyles({
-    link: {
```
Comment (kaankabalak) [inline portal-ui/src/screens/Console/Support/Register.tsx:79]: The `link` class seems to be present on portal-ui/src/screens/Console/Support/Register.tsx:567

File: portal-ui/src/screens/Console/Common/FormComponents/common/styleLibrary.ts [hunk 2]
```diff
@@ -818,23 +634,10 @@ export const wizardCommon = {
...
-  poolError: {
-    color: "#dc1f2e",
-    fontSize: "0.75rem",
-    paddingLeft: 120,
-  },
   error: {
     color: "#dc1f2e",
     fontSize: "0.75rem",
   },
-  h3Section: {
```
Comment (kaankabalak) [inline portal-ui/src/screens/Console/Common/FormComponents/common/styleLibrary.ts:834]: Seems like we use `classes.h3Section` in the following locations:
portal-ui/src/screens/Console/Tenants/AddTenant/Steps/Configure.tsx:295
portal-ui/src/screens/Console/Tenants/AddTenant/Steps/Encryption.tsx:394

**Review by pjuarezd — APPROVED** [review]

**Review by dilverse — APPROVED** [review]

**Review by bexsoft — CHANGES_REQUESTED** [review]
Please confirm that @kaankabalak 's comments doesn't affect any styles

File: portal-ui/src/screens/Console/Support/Register.tsx (hunk 1 above)
Comment (dvaldivia) [reply portal-ui/src/screens/Console/Support/Register.tsx:79]: I removed this on #2630

**Comment by dvaldivia** [comment]
comments addressed, #2630 changes `Register.tsx` and removes links @kaankabalak @bexsoft

**Review by bexsoft — APPROVED** [review]

**Pull request merged**